src/App.tsx

```diff
@@ -1,35 +1,41 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
-import "./App.css";
+import React, { useState } from 'react';
+import './App.css';
+import Button from './components/Button';
 
-function App() {
-  const [count, setCount] = useState(0);
+
+interface Product {
+  id: number;
+  artist: string;
+  title: string;
+  price: number;
+  image: string;
+}
+
+const App: React.FC = () => {
```

Comment:
In newer versions of React (17 and later), the use of React.FC (or React.FunctionComponent) is optional and not required. I think dropping .fc looks a bit cleaner
```suggestion
const App = () => { 
```